newdle/client/src/reducers/creation.js

```diff
@@ -35,6 +36,14 @@ export default combineReducers({
   },
   timeslots: (state = {}, action) => {
     switch (action.type) {
+      case NEWDLE_RECEIVED: {
```

Comment:
Timezone and duration should be handled as well so it doesn't default to the standard values when editing a newdle

Reply:
Thanks! These new fields eventually sneaked in after rebasing.